<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>687</width>
    <height>403</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>151</x>
      <y>32</y>
      <width>277</width>
      <height>202</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QComboBox" name="sceltaTorneo">
       <item>
        <property name="text">
         <string>SitAndGo</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Cash</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayoutNome">
       <item>
        <widget class="QLabel" name="NomeLabel">
         <property name="text">
          <string>Nome</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="NomeLineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayoutNumG">
       <item>
        <widget class="QLabel" name="NumGLabel">
         <property name="text">
          <string>Numero Giocatori</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="NumGLineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayoutImporto">
       <item>
        <widget class="QLabel" name="ImportoLabel">
         <property name="text">
          <string>Importo</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="ImportoLineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayoutBuyIn">
       <item>
        <widget class="QLabel" name="BuyInLabel">
         <property name="text">
          <string>BuyIn</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="BuyInLineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="addTorneo">
       <property name="text">
        <string>Aggiungi</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="ListaTorneiButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>280</y>
      <width>181</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Lista Tornei</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>687</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
